<p>Utiliser la simple c√¥te (') au lieu du guillemet (")</p>
<h2>Noncompliant Code Example</h2>
<pre>
myFunction("nom", "age", "estEtudiant");
  $prenom = "Hugo";
</pre>
<h2>Compliant Solution</h2>
<pre>
myFunction('nom', 'age', 'estEtudiant');
    $prenom = 'Hugo';
</pre>
