<p>Eviter d'utiliser try-catch-finally</p>
<h2>Noncompliant Code Example</h2>
<pre>
        try {
            .....
            }
        } catch (IOException e) {
            .......
        } finally {
            ......
        }
</pre>
<h2>Compliant Solution</h2>
<pre>

        try {
            .....
            }
        } catch (IOException e) {
            .......
        }

</pre>
