<p>Utiliser la simple c√¥te (') au lieu du guillemet (")</p>
<h2>Noncompliant Code Example</h2>
<pre>
print("foo")
myvalue = "foo"

</pre>
<h2>Compliant Solution</h2>
<pre>
print('foo')
myvalue = 'foo'

</pre>
